Source/Core/Core/IOS/FS/FileSystemProxy.cpp

```diff
@@ -29,11 +29,59 @@ static IPCReply GetFSReply(s32 return_value, u64 extra_tb_ticks = 0)
 }
 
 /// Amount of TB ticks required for a superblock write to complete.
-constexpr u64 SUPERBLOCK_WRITE_TICKS = 3370000;
+constexpr u64 GetSuperblockWriteTbTicks(int ios_version)
+{
+  if (ios_version == 28 || ios_version == 80)
+    return 3350000;
+
+  if (ios_version < 28)
+    return 4100000;
+
+  return 3170000;
+}
+
 /// Amount of TB ticks required to write a cluster (for a file).
```

Comment:
Just curious, what is "TB"?

Reply:
I'm guessing timebase?

Reply:
Yep, timebase. I reworded the comments to make that clearer.